src/ui/portal/src/components/Gramercy.vue

```diff
@@ -4,9 +4,10 @@
       <blockquote class="blockquote mb-0">
         <p>{{message}}</p>
         <footer class="blockquote-footer" v-if="!isAnonymous">
-          <a v-bind:href="twitterUrl" target="_blank">{{senderHandle}}</a>
+          <a v-bind:href="senderTwitterUrl" target="_blank">{{senderHandle}}</a>
         </footer>
         <footer class="blockquote-footer" v-else>Anonymous</footer>
+         for <a v-bind:href="recipientTwitterUrl" target="_blank">{{recipientHandle}}</a>
```

Comment:
@Windos you'll kill me, but I'm an idiot.  This is what I get for reviewing PRs when I should be going to bed.

It just dawned on me that your `for ...` snippets need to be inside the `footer` tag. We could probably simplify it like so:

``` html
<footer class="blockquote-footer">
    <a  v-if="!isAnonymous" v-bind:href="senderTwitterUrl" target="_blank">{{senderHandle}}</a>
    <span v-else>Anonymous</span>
     for <a v-bind:href="recipientTwitterUrl" target="_blank">{{recipientHandle}}</a>
</footer>
```
I promise (🤞) to never review a PR when I should be sleeping.

Reply:
All sorted, @MichaelJolley 

Also wrapped a span around the "for ..." as I realised without that it'd probably end up smashed up against the senders twitter handle.